code/_onclick/ai_onclick.dm

```diff
@@ -218,9 +217,8 @@
 	if(isElectrified())
 		electrify(0, user, TRUE) // un-shock
 	else
-		if(user.can_instant_lockdown() || do_after_once(user, 3 SECONDS, target = src, allow_moving = TRUE, attempt_cancel_message = "Shocking [src] cancelled.", special_identifier = "Shock"))
```

Comment:
`can_instant_lockdown()` is only used for these procs and should be removed as well.

Reply:
Removed.